### DISCUSSION TITLE
TTY Terminal Colours

### DISCUSSION THREAD

**Discussion by villaromba** [thehighnibble/thehighnibble.github.io#35] [IMSAI8080esp replica]
Just to kick things off on Github!!!
Suggested possible future enhancement - to have an option for the TTY terminal to have a green/amber/white text option.

**Comment by dmcnaugh** [comment]
Meanwhile you can use standard DEC/ANSI Escape Sequences to change the default colour.
If/when I add a glyph to do this function that is what it will do -ie. inject the Escape Sequences to change the default colour.

**Reply by villaromba** [reply]
Yep, in the meantime, wrote an Mbasic program to select a colour and compiled to a .COM file. Works fine .......
